doc/building_picoquic.md

```diff
@@ -0,0 +1,37 @@
+# Building PicoQUIC
+
+See the 'Building Picoquic' section [README](../README.md) for building picoquic for the first time and installing the dependencies.
+
+## Dependencies
+
+- [Picotls implementation of TLS 1.3](https://github.com/h2o/picotls)
+
```

Comment:
The other dependencies are inherited from picotls. Currently, this means openssl, or a compatible library like libressl. There is work in progress to allow use of mbedtls instead of openssl.

There is a cmake option to build the dependencies as part of building Picoquic. You can see examples of that in the github actions.